caom2pipe/execute_composable.py

```diff
@@ -753,6 +753,79 @@ def execute(self, context):
         self.logger.debug(f'End execute for {self.__class__.__name__}')
 
 
+class MetaUpdateObservationDirect(CaomExecute):
+    """
+    Defines the pipeline step for Collection ingestion of metadata into CAOM.
+    This requires access to only header information.
+
+    This pipeline step will handle the 1 observation ID to n file names
+    relationship when executing fits2caom2, based only on existing
+    metadata from a CAOM2 record.
+    """
+
+    def __init__(self, config, storage_name, command_name, cred_param,
+                 cadc_data_client, caom_repo_client,
+                 observation, meta_visitors, observable):
+        super(MetaUpdateObservationDirect, self).__init__(
+            config, mc.TaskType.INGEST_OBS, storage_name, command_name,
+            cred_param, cadc_data_client, caom_repo_client, meta_visitors,
+            observable)
+        self.observation = observation
+        # set the pre-conditions, as none of this is known, until
+        # the observation is figured out
+        self.fname = None
+        self.lineage = None
+        self.external_urls_param = ''
+        self.logger = logging.getLogger(__name__)
```

Comment:
Typically the logger is declared at the beginning of the file and can then be referred to simply as `logger.debug` for ex.